src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -303,6 +303,7 @@ JitConstants EltwiseKernel_blocked_opt::GetJitConstants(const eltwise_params& pa
     jit.AddConstant(MakeJitConstant("OUTPUT_SIZE_XY", params.outputs[0].X().v * params.outputs[0].Y().v));
     // To calculate batch, define outer block size of feature axis (divided by the inner feature-block size)
     jit.AddConstant(MakeJitConstant("OUT_F_BLOCK", CeilDiv(params.outputs[0].Feature().v, inner_feature_blk_size)));
+    jit.AddConstant(MakeJitConstant("INPUT_F_BLOCK", CeilDiv(params.outputs[0].Feature().v, vec_size)));
```

Comment:
Shouldn't it be like this?
```suggestion
    jit.AddConstant(MakeJitConstant("INPUT_F_BLOCK", CeilDiv(params.inputs[0].Feature().v, vec_size)));
```